app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java

```diff
@@ -201,14 +201,14 @@ public static void setInfoRowDescription(InfoRowView view, String description) {
     //--------------------------------------------------------------
     // SearchResult
     //--------------------------------------------------------------
-    @BindingAdapter("searchResultIcon")
-    public static void setSearchResultIcon(TextView textView, @DrawableRes int iconResId) {
-        Drawable icon = ContextCompat.getDrawable(textView.getContext(), iconResId);
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
-            textView.setCompoundDrawablesRelativeWithIntrinsicBounds(icon, null, null, null);
-        } else {
-            textView.setCompoundDrawablesWithIntrinsicBounds(icon, null, null, null);
-        }
+    @BindingAdapter({"searchResultIcon", "mySession"})
+    public static void setSessionIcon(TextView textView, @DrawableRes int iconResId, boolean isMySession) {
```

Comment:
I commented the icon position on the issue, but now I think it's better to put the icon on the left side 🙇 
No need to fix it on this PR 👍